src/modules/Auth/RequireLoggedIn.js

```diff
@@ -35,13 +35,12 @@ export class RequireLoggedInComponent extends Component {
   }
 
   render() {
-    const styles = require('./Loader.scss');
     if (this.props.authenticated) {
       return <div>{ this.props.children }</div>
     }
     if (this.props.tryingAuth) {
```

Comment:
This might render splash twice - we may only do it in `<TryAuth />`...  I'll experiment.